src/views/HomeView.vue

```diff
@@ -10,4 +11,5 @@
 
 <script setup>
 import Header from '@/components/Header.vue';
+import ProductCard from '../components/ProductCard.vue';
 </script>
```

Comment:
[style] Adicionar linha em branco no final do arquivo
```suggestion
</script>

```